CHANGES.rst

```diff
@@ -1,13 +1,18 @@
 Changelog of fews-3di
 ===================================================
 
-
-2.3 (unreleased)
+2.4 (unreleased)
 ----------------
 
 - Nothing changed yet.
 
 
+2.3 (2023-02-22)
```

Comment:
'em gewoon op "2.3 (unreleased)" houden, want 2.3 is nog niet gereleased.

Later draai ik (of iemand anders) "fullrelease" en die stopt er automatisch de juiste datum in en maakt een nieuwe versieheader aan en past ook de andere plek met het versienummer aan èn maakt een tag. Dus... hoeft niet handmatig :-)

Reply:
Oh, en het is een "breaking change", dus ik wil er een 3.0 i.p.v. 2.3 van maken.

Reply:
Ik heb hem 3.0 (unreleased) genoemd. Gaat dat dan goed?